access/http/models/model_transaction_result.go

```diff
@@ -9,10 +9,11 @@
 package models
 
 type TransactionResult struct {
-	BlockId    string                `json:"block_id"`
-	Execution  *TransactionExecution `json:"execution,omitempty"`
-	Status     *TransactionStatus    `json:"status"`
-	StatusCode int32                 `json:"status_code"`
+	BlockId      string                `json:"block_id"`
+	CollectionId string                `json:"collection_id"`
```

Comment:
Is this part of the PR changes? Does it need testing?

Reply:
This is just taken from https://github.com/onflow/flow-go/blob/master/engine/access/rest/models/model_transaction_result.go.  I had to copy/paste the REST models & GRPC protobufs from flow-go to update to latest version.

The ones in this SDK were outdated and didn't have the `GetNetworkParameters` method.

Reply:
Seems to me you also have to update this function:
https://github.com/onflow/flow-go-sdk/blob/fb5b4a9931490263ede7c8285f517c58fe259682/access/http/convert.go#L357-L374
`TransactionResult` doesn't have `CollectionId` so should consider the gRPC side as well if we're adding.

Reply:
Ah, yeah gotcha.  I'll comb through and deal with all the side effects from updating the REST models.